File: references/MWORKS/MWORKS高校星火计划资料包/培训课程配套材料/01-官网课程配套材料/01-数学与基础应用/03-Julia语法-入门/附件示例/对应课程PPT页码/07_与其他语言的差异/07-2Julia与Python语言的显著差异/page_288/page_288.py
# 定义 class
class Foo():
    baz: int
    qux: float
    def __init__(self, baz, qux):
        self.baz = baz
        self.qux = qux
 # 创建实例
foo = Foo(23, 1.5)
 # 新增字段
foo.abc = 3
foo.abc # 3

File: References/MWORKS/MWORKS高校星火计划资料包/培训课程配套材料/01-官网课程配套材料/01-数学与基础应用/03-Julia语法-入门/附件示例/对应课程PPT页码/07_与其他语言的差异/07-2Julia与Python语言的显著差异/page_288/page_288.jl
# 定义可变结构体
mutable struct Foo
      baz::Int
      qux::Float64
end

# 创建实例
foo = Foo(23, 1.5)
# 不允许增加字段
foo.abc = 3
# ERROR: type Foo has no field abc




